builder/builder.go

```diff
@@ -78,7 +108,7 @@ func (b *Builder) Build() error {
 	carto := mss.New()
 
 	for _, mss := range b.mss {
-		err := carto.ParseFile(mss)
```

Comment:
ParseFile sets the filename so that errors will contain the correct filename, line number and column.

Can you just change https://github.com/omniscale/magnacarto/blob/master/builder/builder.go#L68 to apply baseDir if it is set?